bpf/init.sh

```diff
@@ -292,7 +292,10 @@ function xdp_load()
 	SEC=$6
 	CIDR_MAP=$7
 
-	bpf_compile $IN $OUT obj "$OPTS"
+	NODE_MAC=$(ip link show $DEV | grep ether | awk '{print $2}')
+	NODE_MAC="{.addr=$(mac2array $NODE_MAC)}"
```

Comment:
`NODE_MAC` is already defined in the endpoint's headers, can we reuse that one instead of extending the `init.sh` script? It's easier to debug & introspect that way (not to mention reduces the number of things we'll need to change to eventually get rid of `init.sh`)

Reply:
We already have it for `bpf_load`, but I agree that generally, it might be good. Perhaps that would also make the code generation easier in Martynas' PR. Noting for follow-up.